### ISSUE TITLE
New optional sdr location

### ISSUE THREAD

**Issue by mergen3107** [koreader/contrib#19]
Hi @Hzj-jie ,

There is an ongoing development of the new centralized sdr folder location: https://github.com/koreader/koreader/pull/10132

Just letting you know, in case you would like to update your FindHistory plugin.

**Comment by Hzj-jie** [comment]
Sorry, I totally missed this thread. If my understanding is correct, the new location is configurable. Does it mean setting the home as koreader/ would mitigate the change?

**Comment by Frenzie** [comment]
Pinging @hius07 (since I don't know if you follow this repo)

**Comment by mergen3107** [comment]
Most probably koreeader/docsettings/ , but I don't think it should be set as Home. Home is where all the books are available to browse and open, for example, in Books/

**Comment by Hzj-jie** [comment]
Yes, I am trying to understand if the functionality is totally broken or if there is a way to workaround it. And by default, the locations are still side-by-side with books, right?

**Comment by Frenzie** [comment]
The default hasn't changed. 

I imagine you could work around it (but I'm not entirely sure what it is :-) by reading the relevant location setting or something.

**Comment by hius07** [comment]
We can choose between 3 locations of books metadata: book folder (default),  koreader/docsettings, koreader/hashdocsettings.
Changing the setting doesn't change the location of previously opened books.
This plugin cannot retrieve the book path from two new locations.

**Comment by hius07** [comment]
Now the path to the book is stored in its sdr file as `doc_path`.
